applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -248,6 +248,33 @@ void BaseSolidElement::ResetConstitutiveLaw()
 /***********************************************************************************/
 /***********************************************************************************/
 
+Element::Pointer BaseSolidElement::Clone (
+    IndexType NewId,
+    NodesArrayType const& rThisNodes
+    ) const
+{
+    KRATOS_TRY
+
+    KRATOS_WARNING("BaseSolidElement") << " Call solid base class element Clone " << std::endl;
```

Comment:
why do you have a warning AND an implementation in the Baseclass?

=> Why not throwing if this method is called?

Reply:
Because maybe someone implements a new element and forgets to implement the method. The base element has a warning, not an error

Reply:
ok